### PR TITLE
Circumference Of Circle

### PR DESCRIPTION
**Pull request by TrainedPro**
# Pull Request Template

**Feature Name:** Circumference Of Circle

**Changes made:**<br>

- Added a file which calculates circumference of a circle

### Self Check(Tick After Making pull Request)

- [X] I have read and followed the [Contribution Guidelines](/CONTRIBUTING.md)
- [X] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in the proper folder

- [X] PR is regarding Code
- [x] I am following clean code and Documentation and my code is well linted with flake8.
- [ ] I am using external libraries and I have created requirements.txt in my project folder
- [X] Script is tested and running perfectly fine on my system

- [ ] PR is regarding the documentation

> If the issue was not assigned to you Please don't make a PR, It will be marked as invalid.

Discuss in discord server:- http://pragmaticprogrammer.in/discord

## acknowledgment
- [x] I have tested my code and distributing it under GPL-3.0 License

### CODE REVIEW

File: Math/circumference_of_Circle.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Defining the function to calculate the circumference of a circle
+def Circumference():
+    radius = float(input('Enter Radius Of Circle: '))
+
+    # The formula to calculate circumference
+    circumference = 2*3.14159*radius
```
Comment (Punit-Choudhary) [inline Math/circumference_of_Circle.py:6]: It is better to use PI from the math standard library

Comment (Punit-Choudhary) [reply Math/circumference_of_Circle.py:6]: Also, it would be more readable if you add space between operators.
eg: 2 * PI * radius

Comment (TrainedPro) [reply Math/circumference_of_Circle.py:6]: I avoided using the maths library to make it as default Pythonic as possible, as some people don't want to use certain libraries when there is another way. The number of digits of PI i used in is accurate for quite a bit, even NASA only uses 15 digits.

Secondly, i will definitely fix the code to make it more readible. Is the maths library is a must, also let me know

Comment (Punit-Choudhary) [reply Math/circumference_of_Circle.py]: It was just a recommendation, however, make a static variable for PI

Comment (TrainedPro) [reply Math/circumference_of_Circle.py]: I updated the value of PI to make it static

**Comment by TrainedPro** [comment]
Value of PI has been made static and the code has been improved for readability.

**Pull request closed without merging**